tests/profiles/fvar_test.py

```diff
@@ -447,13 +492,25 @@ def test_check_varfont_distinct_instance_records():
 
     # All of the instance records in the reference varfont are unique
     ttFont = TTFont("data/test/cabinvf/Cabin[wdth,wght].ttf")
-    assert_PASS(check(ttFont), "with a good varfont...")
+    msg = assert_PASS(check(ttFont), "with a good varfont...")
+    assert msg == "All instance records are distinct."
 
     fvar_table = ttFont["fvar"]
     inst_1 = fvar_table.instances[0]
     inst_2 = fvar_table.instances[1]
+    inst_3 = fvar_table.instances[2]
+    inst_4 = fvar_table.instances[3]
 
     # Make instance 2 the same as instance 1
     inst_2.subfamilyNameID = inst_1.subfamilyNameID
     inst_2.coordinates["wght"] = inst_1.coordinates["wght"]
-    assert_results_contain(check(ttFont), WARN, "repeated-instance-records")
+    msg = assert_results_contain(
+        check(ttFont), WARN, "repeated-instance-record:Regular")
```

Comment:
I've seen you using this format of log message IDs that include the `:` and some custom data embedded into it. This is neat, but may make it harder for someone that may want to reuse the check with an override.

Reply:
I borrow the idea from the [com.google.fonts/check/whitespace_glyphs](https://github.com/googlefonts/fontbakery/blob/3e08267464055a9fe6922d4460e7bb47dd74235e/Lib/fontbakery/profiles/universal.py#L397-L398) check. Should I switch to using `-` instead?

Reply:
@felipesanches What's the decision here?

Reply:
no worries. I may propose something else in the future, but I think it is good this way for now.